asset/css/trendingproduct.css

```
@@ -0,0 +1,65 @@
+@font-face {
+   font-family: 'Change Serif';
+   src: url('../asset/fonts/ChangeSerif-SemiBoldItalic.ttf') format('truetype');
+}
+
+.trending-bg-product {
+   background-image: url('../image/bg-product.jpg');
+}
+
+.trending-custom-font .trending-title-font .trending-main-title-font{
+   font-family: 'Playfair Display', serif;
+}
+
+.trending-change-serif {
```

```suggestion
.font-change-serif {
```
Và đặt vào file css tổng quan hơn để dùng lại được nhiều chỗ e nhé, tương tự cho font khác
Cách này tương tự như các framework css